src/braket/aws/aws_device.py

```diff
@@ -747,7 +750,7 @@ def refresh_gate_calibrations(self) -> Optional[GateCalibrations]:
             and self.properties.pulse.nativeGateCalibrationsRef
         ):
             try:
-                with urllib.request.urlopen(
+                with urllib.request.urlopen(  # noqa S310
```

Comment:
Should we add validation here?

Reply:
This comes from the service. We could add the validation but this should already be a trusted source

Reply:
I want to avoid too many inline comments in the code. If later on someone wants to add validation, that would be fine - I am just not inclined to do so as the properties is internally distributed.